src/hub/vendor/customer.py

```diff
@@ -36,6 +39,50 @@ def run(self) -> None:
         self.send_to_routes(routes, json.dumps(data))
 
 
+class StripeCustomerDeleted(AbstractStripeHubEvent):
+    def run(self) -> None:
+        logger.info("customer deleted", payload=self.payload)
+        customer_id = self.payload.data.object.id
+        uid = self.payload.data.object.metadata.userid
+        deleted_user = g.subhub_deleted_users.get_user(uid)
+        result = {}
+        for d in deleted_user.subscription_info:
+            for k in d.keys():
+                result[k] = result.get(k, 0) + d[k]
+        plan_amount = result.get("plan_amount", 0)
+        nicknames = " ".join([x["nickname"] for x in deleted_user.subscription_info])
+        subs = ",".join(
+            [str(x.get("subscription_id")) for x in deleted_user.subscription_info]
```

Comment:
`x["nickname"]` vs `x.get("subscription_id")` any particular reason that these are handled differently?  And why does one coerce to `str`?

Reply:
Good question, applied .get to both and removed str.